.mvn/extensions.xml

```diff
@@ -2,6 +2,6 @@
   <extension>
     <groupId>io.jenkins.tools.incrementals</groupId>
     <artifactId>git-changelist-maven-extension</artifactId>
-    <version>1.0-beta-7</version>
+    <version>1.3</version>
```

Comment:
Fixed Tim, I believe. Thanks for the feedback

Reply:
There’s other stuff that doing a clean incrementalify would do.

I normally delete the maven folder and revert the incremental version change and properties, then it will run again

Reply:
This is fine. Dependabot would prompt you anyway.